src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/personalrelationships/profiledetails/ContactPersonProfileDetailsMigrationService.kt

```diff
@@ -60,35 +61,32 @@ class ContactPersonProfileDetailsMigrationService(
   override suspend fun migrateNomisEntity(context: MigrationContext<PrisonerId>) {
     val prisonerNumber = context.body.offenderNo
     val migrationId = context.migrationId
-    val telemetry = mutableMapOf(
-      "offenderNo" to prisonerNumber,
-      "migrationId" to migrationId,
-      "migrationType" to "CONTACTPERSON_PROFILEDETAILS",
-    )
```

Comment:
I've copied the pattern from ContactPerson where the telemetry is created after the mapping creation succeeds - and the mapping contains all details anyway.